src/commands/deepl.ts

```
@@ -32,8 +34,136 @@ async function deepAuth() {
+            targetLanguage : match[2],
+            name : fileName
+          }
+        }
+      })
+      .filter(glossary => !! glossary) as Array<GlossaryFile>
+  }
+
+  private static getWorkspacePath(): string {
+    try {
+      return workspace.workspaceFolders![0].uri.path
+    } catch (error) {
```

ESLint

[error] Closing curly brace appears on the same line as the subsequent block.

(brace-style)